### PR TITLE
docs: Add maintainer community standards

### PR DESCRIPTION
**Pull request by arbrandes**
### Description

Adds a few explicit community standards for maintainers in the repo maintenance howto.

### CODE REVIEW

File: source/developers/how-tos/maintain-a-repo.rst
Comment (sarina) [inline source/developers/how-tos/maintain-a-repo.rst:176]:
```suggestion
  and screenshots: anybody should be able to understand not only the How,
```
"glean" feels a bit obscure for all the ESL folk in our community

File: source/developers/how-tos/maintain-a-repo.rst
Comment (sarina) [inline source/developers/how-tos/maintain-a-repo.rst:178]:
```suggestion
  a big no-no! Hint: Try viewing your link in an incognito browser to see if it is publicly available.
```

File: source/developers/how-tos/maintain-a-repo.rst
Comment (sarina) [inline source/developers/how-tos/maintain-a-repo.rst:180]:
```suggestion
* Just because you've been entrusted with write access doesn't mean you
```

File: source/developers/how-tos/maintain-a-repo.rst
Comment (sarina) [inline source/developers/how-tos/maintain-a-repo.rst:159]: Should we make Discourse & Slack links? eg https://discuss.openedx.org/ and https://openedx.org/slack

File: source/developers/how-tos/maintain-a-repo.rst
Comment (sarina) [inline source/developers/how-tos/maintain-a-repo.rst:169]:
```suggestion
.. important::
```
I think this may be a better callout!

File: source/developers/how-tos/maintain-a-repo.rst
Comment (sarina) [inline source/developers/how-tos/maintain-a-repo.rst:173]:
```suggestion
   beforehand; even if a feature is by default not shown, experimental code causes cognitive, testing, and potentially even release-impacting burdens.
```

I don't know if this is the right wording, but I think it's important to be explicit about the "why must this be approved"

**Review by sarina — CHANGES_REQUESTED** [review]

**Comment by arbrandes** [comment]
Thanks for the review @sarina!  I've addressed your comments.

**Review by sarina — APPROVED** [review]

**Review by feanil — COMMENTED** [review]
This is a great concise addition to the docs, I think it should go at the top of the ongoing maintainer responsibilities section.  Once it's moved, I think this is good to merge!

**Pull request merged**
